UKHomeOffice/sgmr-service
# SPCR-512 Add noindex and nofollow

### AC / Description of changes
Add noindex nofollow to index.html so it applies across site to prevent search engines indexing and following pages

### To Test
- go to any page
- inspect page
- it should contain `<meta name="robots" content="noindex nofollow" />` as the first element in the `<head>`


### Notes

---

**Remember to merge to feature branches not `master`.**
